Client/Frontend/Home/Pocket/FxHomePocketViewModel.swift

```diff
@@ -79,20 +92,38 @@ class FxHomePocketViewModel {
         TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .pocketStory, value: nil, extras: extras)
     }
 
-    func domainAndReadingTimeForStory(atIndex: Int) -> String {
-        let pocketStory = pocketStories[atIndex]
-        let domainAndReadingTime = "\(pocketStory.domain) • \(String.localizedStringWithFormat(String.FirefoxHomepage.Pocket.NumberOfMinutes, pocketStory.timeToRead))"
-
-        return domainAndReadingTime
-    }
-
     // MARK: - Private
 
     private func getPocketSites() -> Success {
```

Comment:
Could we add tests for the new functions/changes in FxHomePocketViewModel ?

Reply:
Yes. Is it ok to follow up with unit tests in the API integration PR? I'm going refactor the logic a bit.